src/aave-v2/SupplyVaultBase.sol

```diff
@@ -124,15 +131,8 @@ abstract contract SupplyVaultBase is ISupplyVaultBase, ERC4626UpgradeableSafe, O
         override(IERC4626Upgradeable, ERC4626Upgradeable)
         returns (uint256)
     {
-        address poolTokenMem = poolToken;
-        Types.SupplyBalance memory supplyBalance = morpho.supplyBalanceInOf(
-            poolTokenMem,
-            address(this)
-        );
-
-        return
-            supplyBalance.onPool.rayMul(morpho.poolIndexes(poolTokenMem).poolSupplyIndex) +
-            supplyBalance.inP2P.rayMul(morpho.p2pSupplyIndex(poolTokenMem));
+        (, , uint256 totalBalance) = lens.getCurrentSupplyBalanceInOf(poolToken, address(this));
+        return totalBalance;
```

Comment:
Suggestion

```suggestion
        (, , assets) = lens.getCurrentSupplyBalanceInOf(poolToken, address(this));
```
adding `returns (uint256 assets)`

Reply:
I wanted to keep with the openzeppelin format where they do not specify the naming of the return value.

Reply:
Ok for me then